ci-operator/config/cri-o/test/cri-o-test-main.yaml

```
@@ -0,0 +1,40 @@
+base_images:
+  coreos-centos-stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
+  openshift_release_golang-1.17:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+build_root:
+  project_image:
+    dockerfile_path: Dockerfile.crio-test-agent
+images:
+- dockerfile_path: Dockerfile.crio-test-agent
+  from: coreos-centos-stream8
+  inputs:
+    openshift_release_golang-1.17:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.17
+  to: crio-test-agent
+- dockerfile_literal: |
+    FROM base
+    COPY . .
+  from: src
+  to: crio-test-agent-src
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: "echo \"asdf\" \n"
+tests:
+- as: example
+  steps:
+    cluster_profile: packet-assisted
```

```suggestion
    cluster_profile: equinix-ocp-metal
```

otherwise it's billing Ecosystem Engineering organization :sweat_smile:

---

oops!